packages/block-directory/src/plugins/index.js

```diff
@@ -13,6 +13,7 @@ import InstalledBlocksPrePublishPanel from './installed-blocks-pre-publish-panel
 import getInstallMissing from './get-install-missing';
 
 registerPlugin( 'block-directory', {
+	icon: undefined,
```

Comment:
Do you mind adding a small inline comment here so it's not reverted in the future?

Reply:
Makes sense